ref/general/securityUtility.adoc

```diff
@@ -205,13 +205,13 @@ The signature algorithms that end with RSA create certificates with RSA keys.
 The signature algorithms that end with ECDSA create certificates with Elliptical Curve (EC) keys.
 {empty} +
 {empty} +
-If you use certificates that are created with EC keys, then include a customized ciphers list in your TLS configuration to include EC ciphers.
+If you use certificates that are created with EC keys, then include a customized ciphers list in your TLS configuration that includes EC ciphers.
```

Comment:
looking at this again it should probably be

If you use certificates that are created with EC keys, then include a customized ciphers list that includes EC ciphers in your TLS configuration.